packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -72,6 +73,12 @@ const createFluidObject = (async (
         "");
 });
 
+async function ensureContainerConnected(container: Container): Promise<void> {
```

Comment:
It would be nice to have this in a shared location (somewhere in test-utils) rather than duplicated in multiple files. Not a big deal since it's small and already in a few places.